.github/workflows/release.yml

```diff
@@ -56,8 +56,10 @@ jobs:
 
       - name: Archive release
         run: |
-          zip -r lexical.zip _build/dev/rel/lexical
-          cp lexical.zip lexical-${{ steps.timestamp.outputs.result }}.zip
+          cd _build/dev/rel/lexical
+          zip -r lexical.zip *
+          cp lexical.zip ../../../../lexical.zip
```

Comment:
why's this here?

Reply:
Because the step that creates the release expects the ZIPs to be at the root of the directory. It's either that or changing the release step to use `_build/dev/rel/lexical/lexical*.zip` as artifacts.

I figured this was simpler because it avoided having to modify another step, but come to think of it it might make more sense for the ZIPs to be in the same place as the release :thinking: